### DISCUSSION TITLE
Stuck at snapshot download of Goerli testnet

### DISCUSSION THREAD

**Discussion by Vitor-Almeida** [erigontech/erigon#3890] [Q&A] [answered]
Hey, can't tell if this is the right place but,

I've been stuck at the download phase of snapshots for like a week.. been trying to test the Goerli testnet.

![image](https://user-images.githubusercontent.com/23532491/163489234-50348ef2-774b-4837-815b-1efe8c37c4f1.png)

Anyway I can fix this (pic above)? Already tried do restart several times, it just stop downloading at 90% and it never come back.

**Comment by AskAlexSharov** [comment, marked as answer]
Yes, our release of snapshot sync a bit failed due to problems with our infrastructure. Please remove your datadir and start node with --syncmode=fast flag. We will release snapshot sync again next week. Thank you.
